### PR TITLE
Initial book template

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull
React is installed
Linters are installed
Links are added for books, categories and new books
Initial template is setup

### CODE REVIEW

File: src/routes/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function Categories() {
```
Comment (danielufeli) [inline src/routes/Categories.jsx:1]: - [ ] Nice working on your categories page, as a requirement this page is supposed to only display a `button` with the text "Check status", kindly add the "Check status" `button`.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @waqaskanju , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Clean Design✔️
- Professional Repo ✔️
- No Linter Errors ✔️
- Descriptive ReadMe ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Categories.jsx (hunk 1 above)
Comment (waqaskanju) [reply src/routes/Categories.jsx:1]: oh, Sorry I did not see that.
It has been fixed.
image

File: src/routes/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  return (
+    <div>
+      <h1 className="go-down">This is Categories Page</h1>
+      <p>
+        For Now it is empty. Data will follow soon.
+      </p>
+    </div>
+  );
```
Comment (andersonlebon) [inline src/routes/Categories.jsx]: - [ ] Sorry I am not sure you committed and pushed your changes. kindly render the button `check status` as the previous reviewer said.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @waqaskanju 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Categories.jsx (hunk 2 above)
Comment (waqaskanju) [reply src/routes/Categories.jsx]: added
You can check now

**Review by andersonlebon — APPROVED** [review]
## Hi @waqaskanju 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**